P3/ProyectoBase/Src/Engine/Entity.h

```diff
@@ -23,7 +23,13 @@ class Entity {
     // Adds the component to the entity's list of components and to the
     // Component Manager's
     void addComponent(std::string name, Component* c);
+
+	// Returns a component that is expected to exist, throws exception if not
     Component* getComponent(std::string name);
+
+	// Returns the specified component or nullptr if it can't find it
+    Component* findComponent(std::string name);
```

Comment:
Did you run clang-tidy? I'm seeing tabs here.

Reply:
Weird. Maybe it moved during rebase. It is fixed now.